setup.py

```diff
@@ -74,7 +74,7 @@ def recursive_data_files(directory, install_directory):
     "pydocstyle",
     "pyftpdlib",
     "pyinstaller; sys_platform == 'win32'",
-    "pylint",
+    "pylint<3",
```

Comment:
Pylint 3 was released in october and I never noticed!  I assume there are some breaking changes with snapcraft?

Reply:
It's not a huge break, just didn't want to fix linting issues in this PR.